spec/javascripts/test_spec.js

```diff
@@ -51,6 +51,16 @@ describe("Basic testing", function() {
     expect(blurredLocation.getPrecision()).toBe(1);
   });
 
+  it("Checks if getFullLat returns the full latitude of the map", function() {
+    blurredLocation.goTo(45.324324234,-53.32423234234,13);
+    expect(blurredLocation.getFullLat()).toBe(45.324324234);
+  });
+
+  it("Checks if getFullLon returns the full latitude of the map", function() {
+    blurredLocation.goTo(45.324324234,-53.32423234234,13);
```

Comment:
no, it'll be set by zoom, so to set precision, maybe just zoom to a specific level for now. 

Actually we can later add that as a feature -- basically `setZoomByPrecision()` -- good idea. I'll make an issue.